lib/puppet/type/cisco_bgp_af.rb

```diff
@@ -204,8 +204,8 @@ def name
   end
 
   newparam(:afi, namevar: true) do
-    desc 'BGP Address-family AFI (ipv4|ipv6). Valid values are string.'
-    newvalues(:ipv4, :ipv6)
+    desc 'BGP Address-family AFI (ipv4|ipv6|vpnv4|vpnv6). Valid values are string.'
+    newvalues(:ipv4, :ipv6, :vpnv4, :vpnv6)
```

Comment:
It doesn't look like your testing does much with vpnv4/vpnv6 at present - are these sufficiently validated to enable at this time?